tests/show_techsupport/test_auto_techsupport.py

```diff
@@ -129,8 +129,7 @@ def feature_rate_limit_zero(self):
         yield
 
         update_auto_techsupport_feature(self.duthost, self.test_docker,
-                                        rate_limit=DEFAULT_RATE_LIMIT_FEATURE,
-                                        mem_threshold=DEFAULT_AVAILABLE_MEM_THRESHOLD)
+                                        rate_limit=DEFAULT_RATE_LIMIT_FEATURE)
```

Comment:
Is there another caller that will set mem_threshold as a customized value?
If so, the parameter remove could cause a potential failure

Reply:
No, this parameter is added in #9123 by me